test/Console/Command/SuggestCommandTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace PHPSemVerCheckerGit\Test;
+
+use PHPSemVerCheckerGit\Console\Command\SuggestCommand;
+use PHPUnit\Framework\TestCase;
+use PHPSemVerChecker\SemanticVersioning\Level;
+use vierbergenlars\SemVer\version;
+
+class SuggestCommandTest extends TestCase
+{
+    /**
+     * @todo properly mock
+     * @param int $major
+     * @param int $minor
+     * @param int $patch
+     * @return version
+     */
+    private function getMockedVersion($major, $minor, $patch)
+    {
+        return new version("$major.$minor.$patch", true);
+    }
+
+    /**
+     * Provides a few test cases
+     * @return array
+     */
+    public function provideGetNextTag() {
+        return array(
+            //v0.1.0
+            array(Level::NONE, $this->getMockedVersion(0,1,0), '0.1.0'),
+            array(Level::PATCH, $this->getMockedVersion(0,1,0), '0.1.1'),
+            array(Level::MINOR, $this->getMockedVersion(0,1,0), '0.2.0'),
+            array(Level::MAJOR, $this->getMockedVersion(0,1,0), '0.2.0'),
+            //v1.0.0RC1
+            array(Level::NONE, $this->getMockedVersion(1,0,'0-1'), '1.0.0-1'),
+            array(Level::PATCH, $this->getMockedVersion(1,0,'0-1'), '1.0.0-2'),
+            array(Level::MINOR, $this->getMockedVersion(1,0,'0-1'), '1.0.0-2'),
+            array(Level::MAJOR, $this->getMockedVersion(1,0,'0-1'), '1.0.0-2'),
+            //v1.0.0
+            array(Level::NONE, $this->getMockedVersion(1,0,0), '1.0.0'),
+            array(Level::PATCH, $this->getMockedVersion(1,0,0), '1.0.1'),
+            array(Level::MINOR, $this->getMockedVersion(1,0,0), '1.1.0'),
+            array(Level::MAJOR, $this->getMockedVersion(1,0,0), '2.0.0'),
+            //v2.3.4
```

Comment:
Anything in particular not tested by the v1.0.0 case?

Reply:
making sure the incrementing does zero out the next lower version number. Might be a bit more than required.